### PR TITLE
Data visualization container

### PR DESCRIPTION
**Pull request by al-chang**
## Why

[Clickup Ticket](https://app.clickup.com/t/1rp0x7y)

Created the container to hold presentational components for data visualization. 

## This PR

Added a container for the data visualization. Implemented according to the design presented in Figma. This PR does not include wire-up to make calls to back-end, and also does not include adding the routing to this container (not sure if this was desired or not, let me know if I should add it in). 

## Screenshots

Screen Shot 2021-11-05 at 8 59 16 AM
Screen Shot 2021-11-05 at 8 59 23 AM

## Verification Steps

This is mostly a UI implementation, with state set up to track which button is pressed and which drop down value is selected. To test, simply verify that the correct button is selected on the UI after press, and that correct options are presented to the user in the drop down.  I also verified that the drop down value selected updated properly after select.

### CODE REVIEW

**Review by saahilkumar — APPROVED** [review]
Looks good :)

File: src/containers/dataVisualization/index.tsx [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+enum Options {
+  COUNTRY = 'COUNTRY',
+  SCHOOL = 'SCHOOL',
+}
+
+enum Rounded {
+  LEFT = '10px 0 0 10px',
+  RIGHT = '0 10px 10px 0',
+}
```
Comment (maxsebso) [inline src/containers/dataVisualization/index.tsx:19]: These should be placed in a types.ts file in the container folder and then imported

File: src/containers/dataVisualization/index.tsx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+  &:hover,
+  &:focus {
+    background-color: ${(props) => (props.selected ? '#54679E' : '#D4D9E7')};
+    color: ${(props) => (props.selected ? 'white' : 'black')};
+  }
+`;
+
+const DataVisualization: React.FC = () => {
+  const [selectedButton, updateSelectedButton] = useState(Options.COUNTRY);
+  const [schools, updateSchools] = useState<SchoolEntry[]>([]);
```
Comment (maxsebso) [inline src/containers/dataVisualization/index.tsx:49]: don't know if school entry is the best type here. we will decide types when API endpoint is made

File: src/containers/dataVisualization/index.tsx [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+  width: 100%;
+`;
+
+const StyledButton = styled(Button)<{
+  borderradius: string;
+  selected: boolean;
+}>`
+  border-radius: ${(props) => props.borderradius};
+  background-color: ${(props) => (props.selected ? '#54679E' : '#D4D9E7')};
+  color: ${(props) => (props.selected ? 'white' : 'black')};
```
Comment (maxsebso) [inline src/containers/dataVisualization/index.tsx:37]: nice use of props function!

File: src/containers/dataVisualization/index.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  &:focus {
+    background-color: ${(props) => (props.selected ? '#54679E' : '#D4D9E7')};
+    color: ${(props) => (props.selected ? 'white' : 'black')};
+  }
+`;
+
+const DataVisualization: React.FC = () => {
+  const [selectedButton, updateSelectedButton] = useState(Options.COUNTRY);
+  const [schools, updateSchools] = useState<SchoolEntry[]>([]);
+  const [selectedDropDownValue, updateSelectedDropDownValue] = useState<
```
Comment (maxsebso) [inline src/containers/dataVisualization/index.tsx:50]: try to use the naming convention of [thing, setThing] for useStates

File: src/containers/dataVisualization/index.tsx [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+  useEffect(() => {
+    if (availableSchools.kind === AsyncRequestKinds.Completed) {
+      updateSchools(availableSchools.result);
+    }
+  }, [availableSchools]);
+
+  return (
+    <>
+      <Container>
+        <Row style={{ marginBottom: '20px' }} justify="center">
```
Comment (maxsebso) [inline src/containers/dataVisualization/index.tsx:67]: should not use style prop. Instead make a styled component

**Review by maxsebso — CHANGES_REQUESTED** [review]
Looks good! Just a few small changes requested.

**Review by yanDavid21 — APPROVED** [review]
LGTM!

File: src/containers/dataVisualization/index.tsx (hunk 3 above)
Comment (al-chang) [reply src/containers/dataVisualization/index.tsx:32]: Thanks 😄

**Review by maxsebso — APPROVED** [review]

**Pull request merged**